### PR TITLE
Key components readmes

### PR DESCRIPTION
**Pull request by nstsj**
Currently all readmes are build according to this template:
__________
# Component Name

## Description

component_type: *add here*
is_customizable: true/false *(choose one, pls)*

*text for short description here*

## I/O
*text here if i/o specified*

## Dependencies

Configuration settings specified in the .yaml files at [service_configs](link)

Required Python packages specified in [requirements.txt](link).

For more details on inner workings of the component please refer to [server.py](link)
__________

Things to correct/improve: 
- overall readme format for desired points to include there. 
- I/O info on some components (where not specified)

### CODE REVIEW

File: annotators/BadlistedWordsDetector/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Description
+component_type: annotator
+is_customizable: true
+
+Spacy-based user utterance annotator that detects words and phrases from the badlist
+
+## I/O
+input: user input as a str
+output: json dict
```
Comment (dilyararimovna) [inline annotators/BadlistedWordsDetector/README.md:11]: вообще не про typing же это поле, а про смысл - что в этом дикте лежит

File: annotators/BadlistedWordsDetector/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# BadlistedWordsDetector
+
+## Description
+component_type: annotator
+is_customizable: true
```
Comment (dilyararimovna) [inline annotators/BadlistedWordsDetector/README.md:5]: 4 и 5 строки  - это можно не писать

File: annotators/BadlistedWordsDetector/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## I/O
+input: user input as a str
+output: json dict 
+
+## Dependencies
+
+Configuration settings specified in the .yaml files at [service_configs](service_configs)
+Required Python packages specified in [requirements.txt](requirements.txt).
+For more details please refer to [this script](server.py)
```
Comment (dilyararimovna) [inline annotators/BadlistedWordsDetector/README.md:17]: тут имелось в виду, зависит ли от других компонентов в пайплайне.

**Review by dilyararimovna — CHANGES_REQUESTED** [review]

**Comment by nstsj** [comment]
PR closed bc it was a test one, we'll re-write readmes

**Pull request closed without merging**